File: 1.py
num = 3
try:
    n = int(num)
    if n >= 0:
        count = 0
        sum = 0
        while count < n:
            count = count+1
            sum = sum+count
        print("Sum of integers till the input:", sum)
    else:
        nn = -1*n
        count = 0
        sum = 0
        while count < nn:
            count = count+1
            sum = sum+count
        summ = -1*sum
        print("Sum of integers till the input:", summ)
except ValueError:
    print('Type a valid number!')

File: 1.c
#include <stdio.h>

int main() {
    int s=0,m=0,n=0,t=0;
    printf("Integer input:");
    scanf("%d", &n);
   if (n >= 0){
       int i;
        for(i=1;i<=n;++i){
            s +=i;
        }
        printf("Sum of integers till the input:%d", s);
   }
   else{
       m=-1*n;
        int i;
        for(i=1;i<=m;++i){
            s +=i;
        }
        t=-1*s;
        printf("Sum of integers till the input:%d", t);
    }
    return 0;
}
